feed.xml

```diff
@@ -1,5 +1,5 @@
 ---
-layout: none
+layout:
```

Comment:
I just reacted to the warning messages that jekyll was giving me while I was doing this. They seemed to stop when I took away `none`, but if `null` is better go with that.